editions/de-AT-server/tiddlywiki.info

```diff
@@ -2,7 +2,8 @@
 	"description": "Server configuration of the German (Österreich) edition",
 	"plugins": [
 		"tiddlywiki/tiddlyweb",
-		"tiddlywiki/filesystem"
+		"tiddlywiki/filesystem",
+		"tiddlywiki/highlight"
```

Comment:
> As I suggested, I want to move every language edition into its own repo, with there own CI & update mechanism, that is independent to the core. The problem is the current setting for github pages.

Yes, I'm sorry that work stalled; I got rather distracted by the text editor toolbar. Once 5.1.12 is out we should pick this discussion up again.